modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -0,0 +1,120 @@
+
+`name`:: Specify a name for the LVM volume group (VG). 
++
+If this field is set to the name of a VG from the previous {lvms} installation, {lvms} recreates the resources related to that volume group in the current {lvms} installation. You can only recover the VGs but not the logical volume associated with the VG.
++
+[IMPORTANT]
+====
+Before recovering the VGs from the previous {lvms} installation, ensure that the following conditions are met:
+
+* VGs must not be corrupted.
+* VGs must have the `lvms` tag. For more information on adding tags to LVMS objects, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_logical_volumes/grouping-lvm-objects-with-tags_configuring-and-managing-logical-volumes#doc-wrapper[Grouping LVM objects with tags].
+====
+
+`fstype`:: Set this field to `ext4` or `xfs`. By default, this field is set to `xfs`.
+
+`default`:: Set this field to `true` to indicate that a `deviceClass` is the default. Else, you can set it to `false`. You can only configure a single default `deviceClass`. 
+
+`nodeSelector`:: Contains the configuration to choose the nodes on which you want to create the LVM volume group. If this field is empty, all nodes without no-schedule taints are considered.
++
+[NOTE]
+====
+On the control-plane node, {lvms} detects and uses the additional worker nodes when the new nodes become active in the cluster. 
+====
+
+`nodeSelectorTerms`:: Configure a list of requirements that are used to select the node.
+     
+`deviceSelector`:: Contains the configuration to specify the paths to the devices that you want to add to the LVM volume group, and force wipe the devices that are added to the LVM volume group. 
++
+You can specify the device paths in the `paths` field, the `optionalPaths` field, or both. If you do not specify the device paths in both `paths` and `optionalPaths`, {lvms} adds the supported unused devices to the LVM volume group.
++
+[IMPORTANT]
+====
+After a device is added to the LVM volume group, it cannot be removed.
+==== 
++
+{lvms} adds the devices to the LVM volume group only if the following conditions are met:
+
+* The device path exists.
+
+* The device is supported by {lvms}.
++
+[NOTE]
+====
+The devices that you want to add to the volume group must be supported by {lvms}. For information about unsupported devices, see _Devices not supported by {lvms}_ in _Additional resources_.
+====
++
+You can also add the path to the RAID arrays to integrate the RAID arrays with {lvms}. For more information, see _Integrating RAID arrays with {lvms}_ in _Additional resources_. 
+        
+`paths`:: Specify the device paths.
++
+If the device path specified in this field does not exist, or the device is not supported by {lvms}, `LVMCluster` CR creation fails with an error.
+
+`optionalPaths`:: Specify the device paths.
```

Comment:
Device paths can be specified in both `paths` and `optionalPaths`. No restrictions on the type of devices that can be specified here. The only difference is with how the operator responds when it finds an invalid device path. In case of paths, the operator throws an error. In case of optionalPaths, the device is just ignored.

Reply:
Sure. It's fine to leave as is. What I was thinking was that it might be better to be slightly different, like "Specify the optional device paths."